force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +45,10 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        if(age >= 18){
+            return true;
+        }
+        return false; // Replace null with the variable you used to store the result
```

Comment:
See if you can do this one in one line of code with no conditionals. Think about what expressions using < or > evaluate too.